ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_sssd.py::TestSSSDWithAdTrust
         template: *ci-master-latest
         timeout: 3600
         topology: *master_1repl_1client
```

Comment:
@amore17 `test_sssd.py` requires `adroot_adchild_adtree_master_1client` as the topology.